lib/auth/sessions.go

```diff
@@ -368,6 +369,14 @@ func (a *Server) newWebSession(
 			Warn("Failed to calculate trusted device mode for session")
 	} else {
 		sess.SetTrustedDeviceRequirement(tdr)
+
+		if tdr != types.TrustedDeviceRequirement_TRUSTED_DEVICE_REQUIREMENT_UNSPECIFIED {
+			log.WithFields(logrus.Fields{
+				"session":                    sess.GetName(),
+				"user":                       req.User,
```

Comment:
Does this is safe ? My concerns is  that the `req.Users` struct contains fields like req.Users.SSHPrivateKey/req.TLSPrivateKey that probably should not be printed as log output.